### ISSUE TITLE
Simply start acting: Yet another button, not a link

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/accessibility#57]
Hi all. This is the next idea for simple PR. Who will do it?
## Improve accessibility of button Print in Print modal window
In the modal window there is a link **Print**. This link looks like a button, but it's a CSS styling. 

![print](https://user-images.githubusercontent.com/29527200/53293469-6e9eb500-37d4-11e9-9221-f2749dd87cb8.png)

Of course, it should be a button, not a link. 
You need a simple patch in the code. Path to file: administrator/components/com_content/Service/HTML/Icon.php

**Comment by tarot-ray** [comment]
administrator/components/com_content/Service/HTML/Icon.php ??? This path doesn't exist. There is no 'Service' directory under the administrator/components/com_content/ directory.

**Comment by brianteeman** [comment]
@tarot-ray yes there is - are you looking at the wrong branch?

**Comment by tarot-ray** [comment]
@brianteeman Yes, I was. Sorry for that. I made a PR for it.

**Comment by wilsonge** [comment]
This got removed from J4 so nothing left to do here
